flink-contrib/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackend.java

```diff
@@ -356,28 +354,26 @@ public int getKeyGroupPrefixBytes() {
 		final long checkpointTimestamp,
 		final CheckpointStreamFactory checkpointStreamFactory) throws Exception {
 
+		if (db == null) {
+			throw new IOException("RocksDB closed.");
```

Comment:
This exception message looks a bit odd to me. Can you please enrich the message? E.g. "RocksDB has been closed expected. <suggestion for a solution>"?